app/Application.php

```diff
@@ -14,13 +14,16 @@ class Application
 
     public function __construct()
     {
-        $this->messages = $this->loadLocale();
+        $DS = DIRECTORY_SEPARATOR;
+        $lang = file_get_contents("locale{$DS}lang.txt");
```

Comment:
please implement the separate helper `prepare_file_path` which will replace / and \ to the DIRECTORY_SEPARATOR as well